pkg/objectstate/rte/rte.go

```diff
@@ -361,7 +361,6 @@ func UpdateDaemonSetUserImageSettings(ds *appsv1.DaemonSet, userImageSpec, built
 		klog.Warningf("Cannot modify the command line arguments %v", cnt.Args)
 		return nil
 	}
-	fl.SetToggle("--exit-on-conf-change")
```

Comment:
this is correct, but we can and should remove all the `flagcodec` processing here, lines [359, 365]. In any case, let's keep the `flagcodec` package around for some time still (I don't remember if this is the only usage of that package; likely not).